src/components/structures/RightPanel.js

```diff
@@ -91,8 +92,8 @@ module.exports = React.createClass({
         if (MatrixClientPeg.get().isGuest()) {
             var NeedToRegisterDialog = sdk.getComponent("dialogs.NeedToRegisterDialog");
             Modal.createDialog(NeedToRegisterDialog, {
-                title: "Please Register",
-                description: "Guest users can't invite users. Please register to invite."
+                title: counterpart.translate('Please Register'),
+                description: counterpart.translate("Guest users can't invite users. Please register to invite.")
```

Comment:
I know it's easier to use double quotes when there's an apostrophe but https://github.com/matrix-org/matrix-react-sdk/blob/master/code_style.md